Source/EasyNetQ.Management.Client/Model/DeliverGetDetails.cs

```diff
@@ -1,15 +0,0 @@
-﻿using System;
-using EasyNetQ.Management.Client.Serialization;
-using Newtonsoft.Json;
-
-namespace EasyNetQ.Management.Client.Model
-{
-    public class DeliverGetDetails
```

Comment:
Since the DeliverGetDetails, DeliverNoAckDetails and PublishDetails classes all share the same signature and the HTTP API seems to use a common way of returning this data I consolidated them into the MessageRateDetails class instead. Breaking change for anyone depending on the actual class to be there of course. Not sure what your policy is here..